website/static/js/project-organizer.js

```diff
@@ -364,10 +375,14 @@ var tbOptions = {
                     }
                     tb.filter(event);
                 },
-                onchange: m.withAttr('value', tb.filterText),
+                onchange: function() {
+                    m.withAttr('value', tb.filterText);
+                    $osf.trackClick('myProjects', 'filter', 'search-projects');
```

Comment:
Continuation of above.  Onchange does not track every change.  That is one reason why I went with it.  Onchange occurs when the element loses focus, after the content has been changed.